config/vufind/CombinedDriver.ini

```diff
@@ -0,0 +1,36 @@
+[General]
+; Default driver for all methods
+;main_driver = "driver1"
+
+; (Optional) the path to the driver configurations relative to VuFind config
+; dir -- omit to locate those in VuFind config dir -- it also could be an
+; absolute path
+;drivers_config_path = private
+
+; Definition of all the used drivers in the form <DriverName> = <DriverClass>
+; The name of config file for a driver needs te be the same <DriverName>
+[Drivers]
+;driver1 = PAIA
+;driver2 = DAIA
+;driver3 = Demo
+
+; For each driver method it can be specified which drivers should be used and how they can be combined.
+; The head needs to be [<driver method name>]. main_driver specifies the driver that is primarily used.
+; The results of the main driver can be combined with other driver results. Therefore, one needs to specify a
+; merge_key. All results of the support drivers are then matched with the keys of the main driver. E.g. for the
+; method getHolding the results of multiple drivers can get combined if the match on "item_id".
+; The array support_drivers specify the support drivers and which of there keys should be combined with the main result.
+; The format is: support_drivers["<DriverName"] = "<key1>,<key2>,...,<keyN>"
+
+; Here are some examples how a configuration could look like for a combination of DAIA and PAIA
+; but it most likely wont work with the Demo driver
+;[getStatus]
+;main_driver = "driver2"
+;
+;[getStatuses]
+;main_driver = "driver2"
+;
+;[getHolding]
+;main_driver = "driver2"
+;merge_key = "item_id"
```

Comment:
Would it be imaginable that a different merge key would be needed for different support drivers, i.e. should it be specific to each support driver?

Reply:
We don't have a use case for this but I think it makes sense to have the option. I implemented it.